### PR TITLE
feat: Renamed python3.8-11 to python3.12 in examples, added tag to resources

### PR DESCRIPTION
**Pull request by antonbabenko**
- **feat: Added terraform-aws-modules tag**
- **feat: Renamed python3.[8-11] => python3.12 in examples**

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -135,7 +135,11 @@ resource "aws_lambda_function" "this" {
     }
   }
 
-  tags = merge(var.tags, var.function_tags)
+  tags = merge(
```
Comment (antonbabenko) [inline main.tf:138]: @bryantbiggs WDYT about tags? (the rest of the renaming was automatic)

Comment (bryantbiggs) [reply main.tf:138]: The function def, there won't be any insight into the s3 object, cloudwatch could technically be derived but ok either way. Typically, the fewer the better, and tag the main thing which other things can be derived off of that

**Review by bryantbiggs — APPROVED** [review]

File: main.tf (hunk 1 above)
Comment (antonbabenko) [reply main.tf:138]: Updated in 16b854a

**Comment by antonbabenko** [comment]
This PR is included in version 7.5.0 :tada:

**Pull request merged**